app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1500,7 +1500,7 @@ $include-html-global-classes: $include-html-classes;
 }
 
 body {
-  background: #5c5c5c;
+  background: #202020;
```

Comment:
Color literals like `#202020` should only be used in variable declarations; they should be referred to via variable everywhere else.